Purpose: Patch problem reported at 2003-03-10 on
         <URL:http://freshmeat.net/projects/sysvinit/>.
Authour: Petter Reinholdtsen
Fixes:   -
Status:  Should be applied upstream

--- a/src/init.c
+++ b/src/init.c
@@ -2645,9 +2645,10 @@
 	 */
 	isinit = (getpid() == 1);
 	for (f = 1; f < argc; f++) {
-		if (!strcmp(argv[f], "-i") || !strcmp(argv[f], "--init"))
+		if (!strcmp(argv[f], "-i") || !strcmp(argv[f], "--init")) {
 			isinit = 1;
 			break;
+		}
 	}
 	if (!isinit) exit(telinit(p, argc, argv));
 
